bricks-assemble/src/test/java/com/github/ingogriebsch/bricks/assemble/collector/ComponentCollectorTest.java

```diff
@@ -79,29 +78,36 @@ public void collect_should_throw_exception_if_input_is_null() throws Exception {
 
     @Test
     public void collect_throws_exception_if_collector_throws_exception() throws Exception {
+        String applicationId = "applicationId";
+
+        given(collector.collect(applicationId)).willThrow(new RuntimeException());
+
         Assertions.assertThrows(RuntimeException.class, () -> {
-            String applicationId = "applicationId";
-            given(collector.collect(applicationId)).willThrow(new RuntimeException());
             new ComponentCollector(collector, reader).collect(applicationId);
         });
     }
 
     @Test
     public void collect_throws_exception_if_assembler_throws_exception() throws Exception {
+        String applicationId = "applicationId";
+
+        given(collector.collect(applicationId)).willReturn(newHashSet(randomAlphabetic(6)));
+        given(reader.read(anyString())).willThrow(new RuntimeException());
+
         Assertions.assertThrows(RuntimeException.class, () -> {
-            String applicationId = "applicationId";
-            given(collector.collect(applicationId)).willReturn(newHashSet(randomAlphabetic(6)));
-            given(reader.read(anyString())).willThrow(new RuntimeException());
             new ComponentCollector(collector, reader).collect(applicationId);
         });
     }
 
     @Test
     public void collect_should_call_collector() throws Exception {
         String applicationId = "applicationId";
+
```

Comment:
really not trying to be picky, but trying to get your perspective on well formatted code:
why this particular empty line? it is no longer arrange/act/assert separation, is it?

Reply:
Because in my heart I'm a newline fanboy! :P

Reply:
ok, i'll stop this discussion here :p